app/frontend/src/cljs/teet/contract/contract_responsibilities_view.cljs

```diff
@@ -2,63 +2,105 @@
   (:require [teet.common.common-styles :as common-styles]
             [teet.localization :refer [tr tr-enum]]
             [herb.core :refer [<class] :as herb]
-            [teet.ui.icons :as icons]
             [teet.ui.text-field :refer [TextField]]
             [teet.contract.contract-common :as contract-common]
             [teet.contract.contract-style :as contract-style]
             [teet.ui.material-ui :refer [Grid Checkbox Divider]]
             [teet.ui.typography :as typography]
-            [teet.ui.buttons :as buttons]
             [reagent.core :as r]
             teet.contract.contract-spec
-            [teet.common.common-controller :as common-controller]
-            [teet.contract.contract-partners-controller :as contract-partners-controller]
-            [teet.routes :as routes]
-            [teet.ui.form :as form]
-            [teet.ui.select :as select]
-            [teet.ui.common :as common]
-            [teet.ui.validation :as validation]
-            [teet.authorization.authorization-check :as authorization-check]
             [teet.user.user-model :as user-model]
-            [teet.ui.format :as format]
-            [teet.theme.theme-colors :as theme-colors]
-            [teet.ui.table :as table]
-            [clojure.string :as str]
-            [re-svg-icons.feather-icons :as fi]
             [teet.ui.url :as url]
-            [teet.contract.contract-model :as contract-model]
-            [teet.contract.contract-status :as contract-status]))
+            [teet.ui.util :as ui-util]))
 
-(defn pull-tasks [target]
-  ["Task is here" "TRAM person" "Company responsible" "Status"])
+(defn simple-table-with-many-bodies
+  [table-headings groups]
+  [:table {:style {:border-collapse :collapse
+                   :width "100%"}}
+   [:colgroup
+    [:col {:span "1" :style {:width "35%"}}]
+    [:col {:span "1" :style {:width "25%"}}]
+    [:col {:span "1" :style {:width "25%"}}]
+    [:col {:span "1" :style {:width "15%"}}]]
```

Comment:
I'll probably refactor this so that these come from `table-headings` definitions rather than being hard coded in my next PR, but for this PR, this is good.